english/duck/duck.md

```diff
@@ -7,6 +7,8 @@ Make it better next time.
 
 Wait, what was that noise?
 
-Large bats fly over your head.
+Large bats fly over your head. Your hear voices coming from deeper within the cave. It's getting louder and louder. You don't know what to do but you realize you have to make a choice.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.